python/singa/image_tool.py

```diff
@@ -264,6 +295,45 @@ def resize_by_list(self, size_list, num_case=1, inplace=True):
         else:
             return new_imgs
 
+    def resize_by_hw_range(self, rng, inplace=True):
```

Comment:
Yes. resize_by_hw_range and resize_by_hw_list resize images to _(height, width)_, while scale_by_range and scale_by_list scale _the short edge to a small_size_.